clang/lib/CIR/CodeGen/CIRGenCall.cpp

```diff
@@ -889,6 +889,22 @@ RValue CIRGenFunction::emitCall(const CIRGenFunctionInfo &CallInfo,
           auto Results = theCall->getOpResults();
           assert(Results.size() <= 1 && "multiple returns NYI");
           assert(Results[0].getType() == RetCIRTy && "Bitcast support NYI");
+
+          auto reg = builder.getBlock()->getParent();
```

Comment:
Nit: I think `region` would be a clearer name for this variable. `reg` makes me thing of "register" first.